src/ansiblelint/cli.py

```diff
@@ -88,7 +88,8 @@ def load_config(config_file: str | None) -> tuple[dict[Any, Any], str | None]:
 
     config = clean_json(config_lintable.data)
     if not isinstance(config, dict):
-        raise RuntimeError("Schema failed to properly validate the config file.")
+        msg = "Schema failed to properly validate the config file."
+        raise RuntimeError(msg)
```

Comment:
This does not make sense, is a static string. We should use a variable only if we have expansion.